python-spec/src/somacore/query/axis.py

```diff
@@ -28,10 +32,10 @@ def _canonicalize_coords(
 def _canonicalize_coord(coord: options.SparseDFCoord) -> options.SparseDFCoord:
     """Validates a single coordinate, freezing mutable sequences."""
     if coord is None or isinstance(
-        coord, (int, slice, pa.Array, pa.ChunkedArray, np.ndarray)
+        coord, (bytes, int, slice, str, pa.Array, pa.ChunkedArray, np.ndarray)
```

Comment:
nit: put `bytes` next to `str`

Reply:
I considered that, but decided on this since it puts all the built-in types in alphabetical order.